src/helpers.js

```diff
@@ -70,6 +70,13 @@ export const mapActions = normalizeNamespace((namespace, actions) => {
   return res
 })
 
+export const createNamespacedHelpers = (namespace) => ({
+  mapNamespacedState: mapState.bind(null, namespace),
+  mapNamespacedGetters: mapGetters.bind(null, namespace),
+  mapNamespacedMutations: mapMutations.bind(null, namespace),
+  mapNamespacedActions: mapActions.bind(null, namespace)
```

Comment:
I prefer to use the same name as normal `mapXXX` helpers here.
It would be more consistent with the normal helpers and the users can rename them if they needs.